plugins/coral-plugin-offtopic/client/components/OffTopicFilter.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import styles from './styles.css';
+
+export default class OffTopicFilter extends React.Component {
+
+  tag = 'OFF_TOPIC';
+  className = 'offTopicComment';
```

Comment:
Yeah. Well this is a specific request where people using this plugin would like to add a new className. That's one of the features of this PR.